content-repo/extra-docs/releases/20.8.1.md

```diff
@@ -0,0 +1,576 @@
+# Cortex XSOAR Content Release Notes for version 20.8.1 (88914)
+##### Published on 16 August 2020
+
+---
+
+Welcome to the 20.8.1 Content Release for Cortex XSOAR. Starting from the 20.6.0 release, we restructured our release notes to be based on **Content Packs**. One of our team's top priorities is making our Content more accessible and understandable for both users and contributors. In this effort, we recently moved our [Content repo](https://github.com/demisto/content) to work in Pack format, in which there is a clear separation and grouping of Content artifacts. Each Content Pack provides a clear grouping of related Content artifacts used to either implement a use case, implement an integration or provide a clear set of functionality. Our new release notes are structured around Content Packs and you will see related Content artifacts grouped together according to Packs. We hope you will find this new format useful and clear.
+
+#### For Cortex XSOAR version 5.5 and earlier, you can still install content updates directly in the platform. 
+
+---
+
+### New: CyberArk Pack v1.0.0
+#### Integrations
+##### CyberArk PAS  
+Use the CyberArk Privileged Access Security (PAS) solution to manage users, safes, vaults, and accounts from Cortex XSOAR.
+
+
+---
+
+### New: Druva Ransomware Response Pack v1.0.0
+#### Integrations
+##### Druva Ransomware Response  
+Druva Ransomware Response Integration provides ransomware protection for endpoints, SaaS applications and data center workloads for Druva Ransomware Recovery customers.
+
+
+---
+
+### New: Infinipoint Pack v1.0.0
+#### Classifiers
+##### Infinipoint - Classifier  
+Classifies Infinipoint incidents.
+##### Infinipoint  
+
+##### Infinipoint - Incoming Mapper  
+Maps Infinipoint incident fields.
+#### IncidentFields
+- **Infinipoint hostname**
+- **Infinipoint policyID**
+- **Infinipoint policyName**
+#### IncidentTypes
+- **Infinipoint Compliant**
+- **Infinipoint NotCompliant**
+#### Integrations
+##### Infinipoint  
+Use the Infinipoint integration to retrieve security and policy incompliance events, vulnerabilities or incidents. Investigate and respond to events in real-time.
+#### Layouts
+- **Infinipoint Compliant - Summary**
+- **Infinipoint NotCompliant - Summary**
+
+
+---
+
+### New: Mail Listener Pack v1.0.0
+#### Integrations
+##### Mail Listener v2  
+Listens to a mailbox and enables incident triggering via e-mail.
```

Comment:
Let's add the following: We recommend using this integration instead of the built-in Mail Listener integration.